import-export-cli/impl/importAPIPolicy.go

```diff
@@ -0,0 +1,245 @@
+	fileParamName := "file"
+
+	headers := make(map[string]string)
+	if isOAuthToken {
+		headers[utils.HeaderAuthorization] = utils.HeaderValueAuthBearerPrefix + " " + accessToken
+	} else {
+		headers[utils.HeaderAuthorization] = utils.HeaderValueAuthBasicPrefix + " " + accessToken
+	}
+	headers[utils.HeaderAccept] = utils.HeaderValueApplicationJSON
+	headers[utils.HeaderConnection] = utils.HeaderValueKeepAlive
+	return utils.InvokePOSTRequestWithFile(uri, headers, fileParamName, importPath)
+}
+
+// resolveImportFilePath resolves the archive/directory for importing API policy
+// First will resolve in given path, if not found will try to load from exported directory
+func resolvePolicyImportFilePath(file, defaultExportDirectory string) (string, error) {
+	// check current path
+	utils.Logln(utils.LogPrefixInfo + "Resolving for Policy path...")
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		// if the file not in given path it might be inside exported directory
+		utils.Logln(utils.LogPrefixInfo+"Looking for Policy in", defaultExportDirectory)
+		file = filepath.Join(defaultExportDirectory, file)
+		if _, err := os.Stat(file); os.IsNotExist(err) {
+			return "", err
+		}
+	}
+	absPath, err := filepath.Abs(file)
+	if err != nil {
+		return "", err
+	}
+
+	return absPath, nil
+}
+
+// Substitutes environment variables in the project files.
+func replaceEnvVariablesInPolicies(policyFilePath string) error {
+	for _, replacePath := range utils.EnvReplaceFilePaths {
+		absFile := filepath.Join(policyFilePath, replacePath)
+		// check if the path exists. If exists, proceed with processing. Otherwise, continue with the next items
+		if fi, err := os.Stat(absFile); err != nil {
+			if !os.IsNotExist(err) {
+				return err
+			}
+		} else {
+			switch mode := fi.Mode(); {
+			case mode.IsDir():
+				utils.Logln(utils.LogPrefixInfo+"Substituting env variables of files in folder path: ", absFile)
+				if strings.EqualFold(replacePath, utils.InitProjectSequences) {
+					err = utils.EnvSubstituteInFolder(absFile, utils.EnvReplacePoliciesFileExtensions)
+				} else {
+					err = utils.EnvSubstituteInFolder(absFile, nil)
+				}
+			case mode.IsRegular():
+				utils.Logln(utils.LogPrefixInfo+"Substituting env of file: ", absFile)
+				err = utils.EnvSubstituteInFile(absFile, nil)
+			}
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// GetAPIPolicyId Get the ID of an API Policy if available
+// @param accessToken : Token to call the Publisher Rest API
+// @param environment : Environment where API policy needs to be located
+// @param policyName : Name of the API policy
+// @param policyVersion : Version of the API policy
+// @return apiId, error
+func GetAPIPolicyId(accessToken, environment, policyName, policyVersion string) (string, error) {
+	// Unified Search endpoint from the config file to search APIs
```

Comment:
Is this comment correct?

Reply:
Fixed by [6e82281](https://github.com/wso2/product-apim-tooling/pull/924/commits/6e82281c87af32ef11a658fdc70bc37bbf5af147)